src/OrleansAzureUtils/Providers/Streams/AzureQueue/AzureQueueBatchContainer2.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Orleans.CodeGeneration;
+using Orleans.Providers.Streams.AzureQueue;
+using Orleans.Providers.Streams.Common;
+using Orleans.Serialization;
+
+namespace Orleans.Runtime.Host.Providers.Streams.AzureQueue
+{
+    [RegisterSerializer]
+    internal class AzureQueueBatchContainer2 : AzureQueueBatchContainer
```

Comment:
Just a nit pick detail: If we're going to specifically version data can we use naming convention of blag and blargV2 rather than just putting a 2 on it?